docs/user-authentication.txt

```diff
@@ -11,14 +11,115 @@ User authentication
 .. meta::
    :keywords: php framework, odm, code example
 
-If you want to use Laravel's native Auth functionality, register this included
-service provider:
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+odm-long+} to modify the ``User``
+model and configure user authentication.
```

Comment:
Modifying the User class is a technical solution, not the goal of the developer which is "Storing users in MongoDB when using Laravel's native auth  functionality".